app/Notifications/NewUpload.php

```diff
@@ -42,6 +43,33 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     *
+     * @return bool
+     */
+    public function shouldSend(User $notifiable): bool
+    {
```

Comment:
Should have a check to not send if the uploader is anon.

Reply:
Added